ios/map.xcodeproj/project.pbxproj

```diff
@@ -480,12 +480,6 @@
 				GCC_WARN_UNUSED_PARAMETER = NO;
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
-				HEADER_SEARCH_PATHS = (
```

Comment:
@anil-shukla-axway , Move these changes to https://github.com/appcelerator-modules/ti.map/pull/296